src/scanner/commons.rs

```diff
@@ -23,11 +23,10 @@ use crate::{
         kpi_token::KPIToken,
     },
     db::models,
-    defillama::DefiLlamaClient,
     http_client::HttpClient,
     ipfs,
     signer::Signer,
-    specification,
+    specification::{self, Specification},
```

Comment:
Unused `Specification` here.

Reply:
Addressed in 6198090a6645e24d3588b724e1dd3f430605800a